qnexus/client/devices.py

```diff
@@ -73,9 +73,9 @@ def get_all(  # pylint: disable=too-many-positional-arguments
                     backend_name=backend_name,
                     device_name=backend_info["device_name"],
                     nexus_hosted=backendinfolist["is_local"],
-                    backend_info=StoredBackendInfo(
-                        **backend_info
-                    ).to_pytket_backend_info(),
+                    backend_info_dict=StoredBackendInfo(**backend_info)
```

Comment:
Rather than convert this to dict, why not just store a `StoredBackendInfo` on the model and convert it to a pytket one when the attribute is called?

Reply:
Cool yeah I think that would work

Reply:
Change has been made